WMFModel/WMFAnnouncementCollectionViewCell.m

```diff
@@ -49,12 +49,13 @@ - (void)prepareForReuse {
 
 - (void)setImageURL:(NSURL *)imageURL {
     if (imageURL) {
-        [self.imageView wmf_setImageWithURL:imageURL
-                                detectFaces:YES
-                                    failure:^(NSError *_Nonnull error) {
-                                    }
-                                    success:^{
-                                    }];
+        __block UIImage *img = [[UIImage alloc] init];
```

Comment:
Interestingly, if you apply just this chunk to "spike/swift3", the flicker problem seems to mostly go away...